meson_options.txt

```diff
@@ -0,0 +1,48 @@
+option(
+    'enable_exempi',
+    type: 'boolean',
```

Comment:
Please use https://mesonbuild.com/Build-options.html#features for this.

Reply:
thanks for the advice